src/main/java/org/jabref/gui/maintable/MainTable.java

```diff
@@ -123,6 +134,38 @@ public MainTable(MainTableDataModel model, JabRefFrame frame,
         database.getDatabase().registerListener(this);
     }
 
+    /**
+     * This is called, if a user starts typing some characters into the keyboard with focus on main table.
+     * The tableview will scroll to the cell with the same starting column value and typed string
+     * @param sortedColumn The ListView currently focused
+     * @param keyEvent The pressed character
+     */
+
+    private void jumpToSearchKey(TableColumn<BibEntryTableViewModel, ?> sortedColumn, KeyEvent keyEvent) {
+        if (keyEvent.getCharacter() == null || sortedColumn == null) {
+            return;
+        }
+
+        if (System.currentTimeMillis() - lastKeyPressTime < 700) {
+            columnSearchTerm += keyEvent.getCharacter().toLowerCase();
+        } else {
+            columnSearchTerm = keyEvent.getCharacter().toLowerCase();
+        }
+
+        lastKeyPressTime = System.currentTimeMillis();
+
+        this.getItems().stream()
+            .filter(item -> Optional.ofNullable(sortedColumn.getCellObservableValue(item).getValue())
+                                    .map(Object::toString)
+                                    .orElse("")
+                                    .toLowerCase().startsWith(columnSearchTerm))
```

Comment:
Please put `.startsWith` in the next line to have all `.` calls in a separate line.

Reply:
Ok, I wonder should we have all `.` calls in a separate line when we write stream like code in java?